csrankings.csv

```diff
@@ -6016,6 +6016,7 @@ Jacob D. Abernethy,Georgia Institute of Technology,https://www.cc.gatech.edu/~ja
 Jacob D. Furst,DePaul University,https://www.cdm.depaul.edu/about/Pages/People/facultyinfo.aspx?fid=365,NOSCHOLARPAGE
 Jacob Eisenstein,Georgia Institute of Technology,http://www.cc.gatech.edu/~jeisenst,Wb_lnjAAAAAJ
 Jacob Feldman,Rutgers University,https://www.cs.rutgers.edu/people/faculty/affiliated-faculty,jacob@ruccs.rutgers.edu
+Jacob Goldberger,  Bar-Ilan University,  http://www.eng.biu.ac.il/goldbej/, vgzrOK4AAAAJ

```

Comment:
you will need to remove the spurious spaces, but more importantly, the affiliation here is not with the CS department -- this person is not listed here (https://cs.biu.ac.il/en/node/88) and does not obviously meet the inclusion criteria.

Reply:
Dear  Emery Berger,
I'm from Bar-Ilan  EE dept (https://engineering.biu.ac.il/en/content/senior-academic-staff)
I can solely advise PhD students in Computer Science and therefore I meet the inclusion criteria.
Thanks,
Jacob